src/main/java/com/pinterest/secor/common/LogFilePath.java

```diff
@@ -118,6 +118,10 @@ public LogFilePath withPrefix(String prefix) {
         return new LogFilePath(prefix, mTopic, mPartitions, mGeneration, mKafkaPartition, mOffset, mExtension);
     }
 
+    public LogFilePath offsetPath() {
```

Comment:
The 2nd method was added to accommodating a recent refactoring of upload manager.